### PR TITLE
Move sub-classes of "PropertyGrid"  classes to their own files

### PR DESCRIPTION
**Pull request by SergeySmirnov-Akvelon**
## Proposed changes
- Moved sub-classes of "PropertyGrid"  class to their own files
- Fixed unit tests

## Customer Impact
- No 

## Regression? 
- No

## Risk
- Minimal

## Test environment(s) 
- Microsoft Windows [Version 10.0.19041.388]
- .NET Core SDK 6.0.0-alpha.1.21073.5

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/4602)

### CODE REVIEW

**Comment by RussKie** [comment]
MC

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObjectTests.cs [hunk 1]
```diff
@@ -15,7 +15,9 @@ public class PropertyGridAccessibleObjectTests : IClassFixture<ThreadExceptionFi
         public void PropertyGridAccessibleObject_Ctor_Default()
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridAccessibleObjectTests.cs:15]: Please adjust the file name as well.

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridToolStripAccessibleObjectTests.cs [hunk 2]
```diff
@@ -14,7 +14,8 @@ public void PropertyGridToolStripAccessibleObject_Ctor_Default()
         {
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridToolStripAccessibleObjectTests.cs:14]: ditto

File: src/System.Windows.Forms/src/System/Windows/Forms/AutomationMessages.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+#nullable disable
+
+using System.Diagnostics;
+using System.IO;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    internal static class AutomationMessages
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/AutomationMessages.cs:13]: @dreddy-work do you know anything about this class? This looks like MAUI test-related infra.

**Review by RussKie — CHANGES_REQUESTED** [review]

File: src/System.Windows.Forms/src/System/Windows/Forms/AutomationMessages.cs (hunk 3 above)
Comment (dreddy-work) [reply src/System.Windows.Forms/src/System/Windows/Forms/AutomationMessages.cs:13]: No. I never touched it.

**Review by RussKie — APPROVED** [review]

**Pull request merged**